### PR TITLE
[test] Cleanup keypair related test utils

### PR DESCRIPTION
**Pull request by lxfind**
The is the first step to clean up the test-utils crate: the goal is to eventually get rid of all functions there that could have been moved into sui-types, to minimize dependency.
This PR moves all the account/keypair related functions out of test-utils into sui-types

### CODE REVIEW

File: crates/sui-types/src/committee.rs [hunk 1]
```diff
@@ -284,6 +285,24 @@ impl Committee {
             .binary_search_by_key(name, |(a, _)| *a)
             .is_ok()
     }
+
+    /*
+    Testing-only methods
+     */
```
Comment (bmwill) [inline crates/sui-types/src/committee.rs:291]: nit: using block comments like this make it difficult to grep for code in the code base and know if something is live or commented out.

**Review by bmwill — APPROVED** [review]

File: crates/sui-types/src/committee.rs [hunk 2]
```diff
@@ -284,6 +285,24 @@ impl Committee {
             .binary_search_by_key(name, |(a, _)| *a)
             .is_ok()
     }
+
+    // ===== Testing-only methods =====
+
+    /// Generate a simple committee with 4 validators each with equal voting stake of 1.
+    pub fn new_simple_test_committee() -> (Self, Vec<AuthorityKeyPair>) {
```
Comment (joyqvq) [inline crates/sui-types/src/committee.rs:292]: `#[cfg(test)]`?

**Review by joyqvq — APPROVED** [review]
thank you 🙏

File: crates/sui-types/src/committee.rs (hunk 2 above)
Comment (lxfind) [reply crates/sui-types/src/committee.rs:292]: We cannot put #[cfg(test)] because we expect it to be used outside of sui-types crate

**Pull request merged**